business/notifications.js

```diff
@@ -6,6 +6,25 @@ const mq = require('zmq').socket('push');
 
 const delay = config.notificationPoll;
 
+function sendReminderAndUpdateDatabase(reminder, subscriptions) {
+  if (subscriptions.length === 0) {
+    const statusName = 'no-subscription-to-send-to';
```

Comment:
Good idea to check for the length of subscription. I wonder if we couldn't just use "error" ?

Otherwise maybe "error-no-subscription" ?